### ISSUE TITLE
Script failure: toggle block fails on iron door or iron trapdoor

### ISSUE THREAD

**Issue by SlimeDog** [SkriptLang/Skript#2296]
### Server Information
Spigot 1.3.2 final or 1.14.4 latest
Skript 2.4-beta3

### Description

**Toggle block** fails on this script. It works as expected through Skript 2.4-alpha4.
```
on rightclick on iron door:
	toggle the block

on rightclick on iron trapdoor:
	toggle the block
```

### Steps to Reproduce

- Update to Skript 2.4-beta3
- Right-click on iron_door or iron_trapdoor
- Observe that door/trapdoor is not toggled

No in-game or log/console errors.

### Expected Behavior

Door/trapdoor should toggle open/close.

**Comment by FranKusmiruk** [comment]
This could be due to the changes on clicking behaviour, could you try using it on a different event?

**Comment by SlimeDog** [comment]
What would you like me to try? This code fails -- easy to test -- but worked prior to beta3.

**Comment by SlimeDog** [comment]
So it is probably the click event that is not being detected. Good call.

**Comment by SlimeDog** [comment]
Skript 2.4-beta4

Toggle still fails. So far as I can determine, the click appears to be caught -- one can send a message based on it, for example; it's the toggle that doesn't fire.

**Comment by bensku** [comment]
This *was* a click event bug after all. Possible one that only occurs on verbosity very high and above.

**Comment by SlimeDog** [comment]
Confirmed fixed in Skript 2.4-beta5. Thanks.
